src/Common/src/Interop/Windows/Crypt32/Interop.certificates_imports.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
@bartonjs, @stephentoub  I specifically did NOT want to mix types and dll-imports because I have unit tests that have "fake dll imports".  So, I need the structures and constants separate from the dll-imports statements.